### PR TITLE
fix(JsonMapper): register Module JavaTimeModule for parse OffsetDateTime

### PR DESCRIPTION
**Pull request by Oleksiivanov**
register Module JavaTimeModule in zeebeObjectMapper for parse OffsetDateTime.

close:
https://github.com/camunda-community-hub/spring-zeebe/issues/293

### CODE REVIEW

File: connector-runtime/pom.xml [hunk 1]
```diff
@@ -41,6 +41,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```
Comment (berndruecker) [inline connector-runtime/pom.xml:44]: Shouldn't this come as transitive dependency from the starter (and thus not be necessary?)

**Review by berndruecker — COMMENTED** [review]
I can have a look or merge it later today (afternoon). Is that sufficient @Oleksiivanov ?

**Comment by ivanjovic** [comment]
I stumbled across the same problem today, maybe it could be an idea to use the existing Objectmapper from the Spring Context (as it already comes with the missing jackson module).
It would make providing a custom Objectmapper easier (just override the default `Objectmapper` Spring Bean with your custom one and the `JsonMapper` would pick it up automatically). This would look something like the following (this worked like a charm in my demo app):
```java
    @Bean
    public JsonMapper jsonMapper(final ObjectMapper objectMapper) {
        return new ZeebeObjectMapper(objectMapper);
    }
```

Don't know if I missed something but that should make the config more flexible.

**Comment by berndruecker** [comment]
I think @ivanjovic's proposal is a good hint for how Spring Boot is typically handling those situations. So I adjust the code accordingly and will also move the `jackson-datatype-jsr310` dependency to the connector-runtime only - as I would love to leave it to spring-zeebe users to decide what jackson dependencies they actually want to have. Testing this now.

**Comment by berndruecker** [comment]
Thanks again @ivanjovic - your comment was spot on and how it should be done using Spring Boot. I adjusted the PR accordingly and will merge it now.

@Oleksiivanov Maybe you can retest the teams connector once you are back?

**Pull request merged**